src/Components/MinimalCtaBanner.tsx

```diff
@@ -58,7 +58,12 @@ export class MinimalCtaBanner extends React.Component<
               </Sans>
             </a>
             <IconContainer onClick={this.dismissCta.bind(this) as any}>
-              <Icon name="close" color={this.props.textColor} fontSize="16px" />
+              <Icon
+                name="close"
+                color={this.props.textColor}
+                fontSize="16px"
+                style={{ cursor: "pointer" }}
```

Comment:
This and the other change to the CtaBanner story were minor bugfixes left over from early explorations. It's possible we don't even use this and/or should remove the changes from the PR but for now I'm thinking we want to prioritize upcoming reaction migration work so am not addressing unless you think it is neccessary @damassi